packages/apps/human-app/server/src/app.controller.ts

```diff
@@ -1,12 +1,32 @@
-import { Controller, Get } from '@nestjs/common';
-import { AppService } from './app.service';
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  Redirect,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { ApiExcludeEndpoint, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { SignupWorkerDto } from './interfaces/signup-worker-request.dto';
+import { AuthWorkerService } from './modules/auth-worker/auth-worker.service';
 
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
+  constructor(private readonly authWorkerService: AuthWorkerService) {}
 
-  @Get()
-  getHello(): string {
-    return this.appService.getHello();
+  @Get('/')
+  @Redirect('/swagger', 301)
+  @ApiExcludeEndpoint()
+  public swagger(): string {
+    return 'OK';
+  }
+
+  @ApiTags('Auth')
+  @Post('/auth/signup')
+  @ApiOperation({ summary: 'Worker registration' })
+  @UsePipes(new ValidationPipe())
+  public signupWorker(@Body() signupWorkerDto: SignupWorkerDto): Promise<void> {
```

Comment:
Don't we need origin restriction on this endpoint? @BartoszSolkaBD

Reply:
probably not necessary for now